cypress/integration/buttons.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { checkEyes, setup } from '../util';
+
+const test = 'Buttons';
+
+// TODO: Enable these tests again after validation
+export function ButtonSpecs() {
+  xdescribe(test, () => {
```

Comment:
Should this suite be skipped?

Reply:
Yes, I wanted to leave the test suite but not run it for now.